src/Cns.ts

```diff
@@ -42,26 +96,38 @@ export default class Cns extends EthereumNamingService {
   isSupportedDomain(domain: string): boolean {
     return (
       domain === 'crypto' ||
-      (domain.indexOf('.') > 0 &&
-        /^.{1,}\.(crypto)$/.test(domain) &&
-        domain.split('.').every(v => !!v.length))
+          (domain.indexOf('.') > 0 &&
+            /^.{1,}\.(crypto)$/.test(domain) &&
+            domain.split('.').every(v => !!v.length))
     );
   }
+  
+  async owner(domain: string): Promise<string> {
+    return (await this.getVerifiedData(domain)).owner;
+  }
 
   async resolver(domain: string): Promise<string> {
     return (await this.getVerifiedData(domain)).resolver;
   }
 
-  async owner(domain: string): Promise<string> {
-    return (await this.getVerifiedData(domain)).owner;
+  async record(domain: string, key: string): Promise<string> {
+    const returnee = (await this.records(domain, [key]))[key];
+    if (!returnee) {
+      throw new ResolutionError(ResolutionErrorCode.RecordNotFound, {recordName: key, domain});
+    }
```

Comment:
I've unresolved this because I noticed that we do `Promise<... | undefined>` for some of the ENS methods already. So I'm curious why we don't want to change this behaviour for the 2.0 release.